### ISSUE TITLE
Fix logging

### ISSUE THREAD

**Issue by parberge** [codelabsab/timereport-api#10]
Right now the logging setup doesn't really work.

We need to do something like this:

In app.py:
```
log_level = logging.DEBUG if app.debug else logging.INFO
app.log.setLevel(log_level)
```

And then in other libs we can do:
```
log = logging.getLogger(f'timereport_backend.{__name__}')
```
Or whatever the app name is defined to (i.e `app = Chalice(app_name='timereport_backend')`)

Chalice setup a logger with that name you specify in app_name.

**Comment by parberge** [comment]
Fixed in b5550994231d0daf0aaddda775a2a3e501b973b3

@kamigerami @tommymalmqvist Can you review the commit? Didn't mean to master but it should not have broken anything ™️ 😄

**Comment by parberge** [comment]
Rolled out with release `0.6.2`
